src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const litleDiscount = 20;
+  const bigDiscount = 50;
+  let amount = 0;
+  const priceMultDays = price * days;
+
+  if (days <= 2) {
+    return amount + priceMultDays;
+  }
+
+  if (days >= 3 && days <= 6) {
+    amount += priceMultDays - litleDiscount;
+
+    return amount;
+  } else if (days >= 7) {
+    amount += priceMultDays - bigDiscount;
+
+    return amount;
+  }
```

Comment:
- if there are `return` inside if block - no need to write `else` after it
- 3 and 7 also should be stored in variables